### PR TITLE
Fix currently trading projects function

### PR DESCRIPTION
**Pull request by IvanIvanoff**
Usage of or_where was completely undermining the other checks. It had to
be replaced with `AND where(X or Y or Z). This was making projects with
is_hidden=true to be included

#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/model/project/list/list.ex [hunk 1]
```diff
@@ -411,7 +414,7 @@ defmodule Sanbase.Model.Project.List do
   one or all of its name, ticker and slug are present int he trending words.
   Filtering out projects based on some conditions can be controled by the options.
 
-  See the "Shared options" section at the module documentation for more options.
+  See the "Shared options" sectiorn at the module documentation for more options.
```
Comment (spiderjako) [inline lib/sanbase/model/project/list/list.ex:417]: Unnecessary change I presume

Comment (IvanIvanoff) [reply lib/sanbase/model/project/list/list.ex]: fixed

**Comment by tspenov** [comment]
yeah, `or_where` and composed queries doesn't work well together
